profiles/acrossx.jsonld

```diff
@@ -1,8 +1,8 @@
 {
-  "@context": "http://xapi.vocab.pub/profiles/profile-context.jsonld",
+  "@context": "https://w3id.org/xapi/profiles/context",
   "id": "https://w3id.org/xapi/acrossx",
   "type": "Profile",
-  "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+  "conformsTo": "https://github.com/adlnet/xapi-profiles/",
```

Comment:
This is not the correct URI. Per the spec, conformsTo must be:  https://w3id.org/xapi/profiles#1.0